src/test/deploy/functions/release/fabricator.spec.ts

```diff
@@ -938,141 +845,6 @@ describe("Fabricator", () => {
-                    limits: {
-                      memory: "256M",
-                      cpu: "0.1667",
-                    },
-                  },
-                },
-              ],
-            },
-          },
-          traffic: [],
-        },
-      };
-    });
-
-    it("replaces the service to set concurrency", async () => {
-      run.getService.resolves(service);
-      run.updateService.resolves(service);
-      await fab.setRunTraits(
-        service.metadata.name,
-        endpoint(
-          { httpsTrigger: {} },
-          {
-            cpu: 1,
-            concurrency: 80,
-          }
-        )
-      );
-      expect(run.updateService).to.have.been.called;
-    });
-
-    it("replaces the service to set CPU", async () => {
-      service.spec.template.spec.containers[0].resources.limits.cpu = (1 / 6).toString();
-      run.getService.resolves(service);
-      run.updateService.resolves(service);
-      await fab.setRunTraits(
-        service.metadata.name,
-        endpoint(
-          { httpsTrigger: {} },
-          {
-            cpu: 1,
-            concurrency: 1,
-          }
-        )
-      );
-      expect(run.updateService).to.have.been.called;
-    });
-
-    it("doesn't setRunTraits when already at the correct value", async () => {
-      service.spec.template.spec.containerConcurrency = 80;
-      service.spec.template.spec.containers[0].resources.limits.cpu = "1";
-      run.getService.resolves(service);
-
-      await fab.setRunTraits(
-        "service",
-        endpoint(
-          {
-            httpsTrigger: {},
-          },
-          {
-            cpu: 1,
-            concurrency: 80,
-          }
-        )
-      );
-      expect(run.updateService).to.not.have.been.called;
-    });
-
-    it("wraps errors", async () => {
-      run.getService.rejects(new Error("Oh noes!"));
-
-      await expect(fab.setRunTraits("service", endpoint())).to.eventually.be.rejectedWith(
-        reporter.DeploymentError,
-        "set concurrency"
-      );
-
-      run.getService.resolves(service);
-      run.replaceService.rejects(new Error("read only"));
-      await expect(fab.setRunTraits("service", endpoint())).to.eventually.be.rejectedWith(
-        reporter.DeploymentError,
-        "set concurrency"
-      );
-    });
-  });
-
-  describe("upsertScheduleV1", () => {
-    const ep = endpoint({
-      scheduleTrigger: {
-        schedule: "every 5 minutes",
-      },
-    }) as backend.Endpoint & backend.ScheduleTriggered;
-
-    it("upserts schedules", async () => {
-      scheduler.createOrReplaceJob.resolves();
-      await fab.upsertScheduleV1(ep);
-      expect(scheduler.createOrReplaceJob).to.have.been.called;
-    });
-
-    it("wraps errors", async () => {
-      scheduler.createOrReplaceJob.rejects(new Error("Fail"));
-      await expect(fab.upsertScheduleV1(ep)).to.eventually.be.rejectedWith(
-        reporter.DeploymentError,
-        "upsert schedule"
-      );
-    });
-  });
-
```

Comment:
don't think you mean't to remove this test.

Reply:
whoops, thanks for catching that!